bazel/cc_toolchains/clang_cc_toolchain_config.bzl

```diff
@@ -291,6 +291,32 @@ def _impl(ctx):
         ],
     )
 
+    x86_64_cpu_flags = feature(
+        name = "x86_64_cpu_flags",
+        enabled = True,
+        flag_sets = [
+            flag_set(
+                actions = all_compile_actions,
+                flag_groups = [flag_group(flags = [
+                    "-march=x86-64-v2",
+                ])],
+            ),
+        ],
+    )
+
+    aarch64_cpu_flags = feature(
+        name = "aarch64_cpu_flags",
+        enabled = True,
+        flag_sets = [
+            flag_set(
+                actions = all_compile_actions,
+                flag_groups = [flag_group(flags = [
+                    "-march=armv8.2-a",
```

Comment:
If we're targetting specific `arch` vs `native`, I feel this could arbitrarily break folks' builds. I'd have to check the "coverage" that `x86-64-v2` and `armv8.2-a` give us, but I think we will make it incompatible with some systems. In which case, we may need to:
* Add a disclaimer/note somewhere
* ... or an understandable error
* ... or a simple way to disable cpu-specific optimizations (manually or automatically)

My fear is that potential new contributors would attempt to build, see a hard to understand error, and get discouraged. But I yet have to check the probability of that happening.